memconfig.lock

```diff
@@ -284,20 +284,6 @@
                 ],
                 "hints": {}
               },
-              {
-                "name": "scoped",
-                "description": "",
-                "type": "Root",
-                "params": [
-                  {
-                    "name": "url",
-                    "description": "",
-                    "type": "String",
-                    "optional": false
-                  }
-                ],
-                "hints": {}
-              },
```

Comment:
For context, this is an experiment Justin and I were working on to make an HTTP node that is scoped to a specific URL, but it's not in http yet, so your removal of it is correct.